test/e2e_node/eviction_test.go

```diff
@@ -576,10 +617,36 @@ func runEvictionTest(f *framework.Framework, pressureTimeout time.Duration, expe
 			time.Sleep(30 * time.Second)
 			ginkgo.By("setting up pods to be used by tests")
 			pods := []*v1.Pod{}
+			podsToSucceed := []*v1.Pod{}
 			for _, spec := range testSpecs {
+				if isPodExpectedToSucceed(spec.pod) {
+					podsToSucceed = append(podsToSucceed, spec.pod)
+					continue
+				}
 				pods = append(pods, spec.pod)
 			}
-			e2epod.NewPodClient(f).CreateBatch(ctx, pods)
+			e2ePodClient := e2epod.NewPodClient(f)
+			e2ePodClient.CreateBatch(ctx, podsToSucceed)
+
+			// Wait for the expected succeeded pods to get to that state
+			for _, podToSucceed := range podsToSucceed {
+				ginkgo.By(fmt.Sprintf("Waiting for the %s pod to complete", podToSucceed.Name))
+				gomega.Eventually(ctx, func() error {
```

Comment:
Use `framework.ExpectNoError(e2epod.WaitForPodSuccessInNamespace(ctx, f.ClientSet, podToSucceed.Name, podToSucceed.Namespace))` instead

Reply:
Done, thank you.